web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -681,8 +681,8 @@ public static void shutdownModules(ServletContext servletContext) {
 		String realPath = getRealPath(servletContext);
 		
 		// clear the module messages
-		String messagesPath = realPath + "/WEB-INF/";
-		File folder = new File(messagesPath.replace("/", File.separator));
+		String messagesPath = Paths.get(realPath,"WEB-INF").toString();
+		File folder = new File(messagesPath);
```

Comment:
same here can you remove the temp variable `messagesPath` by calling `toFile()` on the `Paths.get()`